setup.py

```diff
@@ -26,7 +27,8 @@
     'evm-extra': [
         "coincurve>=7.0.0,<8.0.0",
         "plyvel==1.0.4",
-        "eth-hash[pycryptodome]",
+        "eth-hash[pysha3];implementation_name=='cpython'",
+        "eth-hash[pycryptodome];implementation_nam=='pypy'",
```

Comment:
s/nam/name/

Reply:
@tuxxy, this is not `nam`, this is py-evm - there are rules.